dapp/src/components/buySell/BuySellWidget.js

```diff
@@ -112,6 +118,22 @@ const BuySellWidget = ({
     setBuyFormErrors(newFormErrors)
   }, [dai, usdt, usdc, pendingMintTransactions])
 
+  const setPriceToleranceValue = (value) => {
+    setPriceTolerFct(value)
+    localStorage.setItem(priceToleranceLocalStorageKey, value)
+  }
+
+  // store price tolerance value for future default setting
+  useEffect(() => {
+    // default price tolerance value
+    let priceTolerance = 0.5
+    let localStorageValue = localStorage.getItem(priceToleranceLocalStorageKey)
+    if (localStorageValue) {
+      priceTolerance = parseFloat(localStorageValue)
```

Comment:
nit: A `NaN`, `null` or `undefined` check would be great if you want to be super-defensive.

Reply:
Good thinking thanks